### PR TITLE
Update indicators again.js

### PR DESCRIPTION
**Pull request by jamesbroadbent202**

### CODE REVIEW

File: src/constants/categories.js [hunk 1]
```diff
@@ -635,14 +635,14 @@ export const CATEGORIES = [
           {
             name: 'Energy efficiency of buildings',
             axisTitle: 'Average energy efficiency rating of commercial buildings',
-            description: 'This shows the average National Australian Built Environment Rating System (NABERS) score for rated office buildings in the city, weighted by rated floor space. *Cities with fewer than 10 rated buildings are Albury Wodonga, Ballarat, Bendigo, Cairns, Geelong, Launceston, Mackay, Toowoomba, Townsville, Western Sydney and Sunshine Coast. \n\nSource: National Australian Built Environment Rating System',
+            description: 'This shows the average National Australian Built Environment Rating System (NABERS) score for rated office buildings in the city, weighted by rated floor space. /n/nCities with fewer than 10 rated buildings are Albury Wodonga, Ballarat, Bendigo, Cairns, Geelong, Launceston, Mackay, Toowoomba, Townsville, Western Sydney and Sunshine Coast. \n\nSource: National Australian Built Environment Rating System',
```
Comment (davidgilbertson) [inline src/constants/categories.js]: These slashes are around the wrong way, `/n/nCities` should be `\n\nCities`

File: src/constants/categories.js [hunk 2]
```diff
@@ -826,7 +826,7 @@ export const CATEGORIES = [
         charts: [
           {
             name: 'Workers in knowledge intensive services',
-            axisTitle: 'Percentage of employment as knowledge workers',
+            axisTitle: 'Percentage of employment',
```
Comment (davidgilbertson) [inline src/constants/categories.js]: This axis title is "Percentage of employment" but the similar chart on the jobs page "Industry sector share of employment" has the axis title of "Proportion of employed persons". Should they be the same as each other?

I'd say it's a percentage, not a proportion, and I don't know if we're standardising on 'people' or 'persons'.

File: src/constants/indicators.js [hunk 3]
```diff
@@ -484,8 +484,8 @@ export default {
     source: 'Population-weighted density, Persons per square kilometre',
   },
   dependencyRatioNotWorkingAge: {
-    name: 'Population share not of working age',
-    format: FORMATS.PERCENT_ZERO_OR_ONE_DECIMAL,
+    name: 'Dependency ratio',
+    format: FORMATS.ONE_OR_TWO_DECIMALS,
```
Comment (davidgilbertson) [inline src/constants/indicators.js]: This seems odd. This indicator shows the proportion of (under 14 + over 65) / everyone else, and is _not_ a percentage. But the previous chart on the screen shows the same data, but shows each age group a proportion of the total and that _is_ a percentage?

They logically have to be the same units, right? Or am I missing something?

(top right and bottom left)

image

**Review by davidgilbertson — DISMISSED** [review]

**Comment by jonalport** [comment]
@jamesbroadbent202 - do you know about squashing? 😄

**Review by micapam — APPROVED** [review]
lgtm

**Pull request merged**